packages/unchained-client/src/parser/thormaya.ts

```diff
@@ -0,0 +1,320 @@
+import type { Axios } from 'axios'
+import axios from 'axios'
+
+import type { BaseTxMetadata, Dex, StandardTx } from '../types'
+import { TradeType } from '../types'
+
+type LiquidityType = 'Savers' | 'LP' | 'RUNEPool'
+type SwapType = 'Standard' | 'Streaming'
+
+interface Liquidity {
+  type: LiquidityType
+}
+
+export interface Swap {
+  type: SwapType
+}
+
+interface Coin {
+  amount: string
+  asset: string
+}
+
+interface InOut {
+  txID: string
+  address: string
+  coins: Coin[]
+  height?: string
+}
+
+interface StreamingSwapMeta {
+  count: string
+  depositedCoin: Coin
+  inCoin: Coin
+  interval: string
+  lastHeight: string
+  outCoin: Coin
+  quantity: string
+}
+
+interface Metadata {
+  addLiquidity?: {
+    liquidityUnits: string
+  }
+  refund?: {
+    affiliateAddress: string
+    affiliateFee: string
+    memo: string
+    networkFees: Coin[]
+    reason: string
+  }
+  swap?: {
+    affiliateAddress: string
+    affiliateFee: string
+    isStreamingSwap: boolean
+    liquidityFee: string
+    memo: string
+    networkFees: Coin[]
+    streamingSwapMeta?: StreamingSwapMeta
+    swapSlip: string
+    swapTarget: string
+  }
+  withdraw?: {
+    asymmetry: string
+    basisPoints: string
+    impermanentLossProtection: string
+    liquidityUnits: string
+    memo: string
+    networkFees: Coin[]
+  }
+}
+
+interface Actions {
+  date: string
+  height: string
+  in: InOut[]
+  out: InOut[]
+  metadata?: Metadata
+  pools: string[]
+  status: 'success' | 'pending'
+  type: 'swap' | 'addLiquidity' | 'withdraw' | 'donate' | 'refund' | 'switch' | 'send'
```

Comment:
q: not introduced by this PR, but out of these we're only handling `swap` (`swapOut`) and `withdraw` (`withdrawOut`). 
As far as I can see, none of the others seem to be handled. 
Do we want to follow-up with missing action type handling, or perhaps, as a starter, remove `donate` and `switch` from here which I reckon we'll never handle? 

<img width="808" alt="image" src="https://github.com/user-attachments/assets/fbcb4296-5f05-44fc-9b10-9efa85b5ed66" />
<img width="799" alt="image" src="https://github.com/user-attachments/assets/d5e1432d-c59c-425b-9dcf-370c51167e46" />